src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -785,7 +785,7 @@ export default class DefaultAudioVideoController
           this.configuration.connectionTimeoutMs
         ),
       ];
-
+      this.cleanUpMediaStreamsAfterStop();
```

Comment:
So we just remove the observers and does that help cleaning up the streams as well? Or cleaning up means setting active audio-video input devices to undefined.